### PR TITLE
package/libspatialite_add_v5.0.1

### PR DESCRIPTION
**Pull request by Sinan81**
depends on librttopo PR to be pushed soon.

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
Confirmed the sha256.  Waiting on merge of new dependency package before able to merge this one, which will require this PR be rebased to `develop`.

**Comment by Sinan81** [comment]
can this be merged?

**Review by alalazo — APPROVED** [review]

**Pull request merged**
